From 96d34791ec3f3053bdcccabe7fd022be54c8abd3 Mon Sep 17 00:00:00 2001
From: lelign <ignatlelekin@gmail.com>
Date: Tue, 31 Mar 2026 09:55:03 +0300
Subject: [PATCH] change_gpio_board_config_to_533
Upstream-Status: Pending

---
 board_config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board_config.h b/board_config.h
index 35fe335..1133589 100644
--- a/board_config.h
+++ b/board_config.h
@@ -1,7 +1,7 @@
 #ifndef BOARD_CONFIG_H
 #define BOARD_CONFIG_H
 #if (BOARD_REV==0)
-    #define M26_GPIO "gpio446"
+    #define M26_GPIO "gpio533"
 #else
     #define M26_GPIO "gpio478"
 #endif
